### PR TITLE
Cb 27 

### PR DESCRIPTION
**Pull request by CalebMacdonaldBlack**

### CODE REVIEW

File: src/cljc/centrebull/spec.cljc [hunk 1]
```diff
@@ -96,6 +96,10 @@
     :opt [:shooter/preferred-name
           :shooter/club]))
 
+(s/def ::range-id-only
+  (s/keys
+    :req [:range/id]))
```
Comment (axrs) [inline src/cljc/centrebull/spec.cljc:102]: There's no spec defined for :range/id `(s/def :range/id is-uuid?)`

File: test/clj/centrebull/test/ranges/core.clj [hunk 2]
```diff
@@ -12,5 +13,11 @@
       (with-redefs [dao/create! (mock-dao/create! expected nil)]
         (let [{:keys [status body]} (ranges/create! {:all-params expected})]
           (is (= body nil))
+          (is (= status 200))))))
+  (testing "ranges-delete!"
```
Comment (axrs) [inline test/clj/centrebull/test/ranges/core.clj]: Add a new line above this. Seperates the tests making it a tad more readable.

File: test/clj/centrebull/test/ranges/routes.clj [hunk 3]
```diff
@@ -21,4 +21,9 @@
     (with-redefs [ranges/create! (constantly (response/ok {:range-create! "called"}))]
       (let [{:keys [status body]} ((app) (json-request :post "/ranges" (gen-range)))]
         (is (= status 200))
-        (is (= {:range-create! "called"} (parse-body body)))))))
+        (is (= {:range-create! "called"} (parse-body body))))))
+  (testing "Delete Range Route"
```
Comment (axrs) [inline test/clj/centrebull/test/ranges/routes.clj]: New line above for readability

**Review by axrs — CHANGES_REQUESTED** [review]

**Comment by CalebMacdonaldBlack** [comment]
@axrs

**Review by axrs — APPROVED** [review]

**Pull request merged**